README.md

```diff
@@ -88,6 +94,10 @@ According to the [CPC Charter](https://github.com/openjs-foundation/cross-projec
 - Christopher Hiller (@boneskull, IBM)
 - Tobie Langel (@tobie, UnlockOpen)
 
+#### Collaboration space representatives
+
+Members of the Collaboration spaces at the `Core` stage may nominate a candiate for one of the voting seats on the CPC which represents the Collaboration spaces. Currently there are no spaces at the `Core` stage and therefore no representative.
```

Comment:
What is the `Core` phase? Where is the documentation for this?

Reply:
These are documented here https://github.com/openjs-foundation/cross-project-council/blob/master/proposals/stage-0/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md#core-collaboration-spaces